project_national/housing_characteristics/Corridor.tsv

```diff
@@ -1,7 +1,3 @@
-Dependency=Geometry Building Type RECS	Option=Not Applicable	Option=Double-Loaded Interior	Option=None	Option=Single Exterior Front	Option=Double Exterior
```

Comment:
We should keep the dependency with Geometry Building Type RECS. It doesn't cause an error but stops the results.csv saying that a Mobile Home, Single-Family Detached, and Single-Family Attached have double-loaded corridors.

Reply:
@afontani Same question as Ceiling Fan.

Reply:
We should copy the "Corridor.tsv" from resstock to resstock-estimation.